geometry_msgs/Point[] left_cones
geometry_msgs/Point[] right_cones